### DISCUSSION TITLE
Is there a way to set up a reverse proxy API?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#27699] [Questions]
I need to use nginx reverse proxy implementation on the server, can nuxt be implemented?

![image](https://github.com/nuxt/nuxt/assets/26525599/3c9980ae-1e63-48a1-949a-a1bd6af158bf)

**Comment by TheAlexLichter** [comment]
See [this video](https://www.youtube.com/watch?v=J4E5uYz5AY8) for thorough info on proxying an API :)

**Reply by bbhxwl** [reply]
I watched the video and it can be set up like this, but if you want to distinguish between development and formal deployment, how should you set up different APIs?


```
    vite:{
        server:{
            proxy:{
                '/Api':{
                    target:'https://api.***.com/Api/ADLWebApi/v3/',
                    changeOrigin:true,
                    rewrite:(path)=>{
                        console.log("path",path)
                        return path.replace(/^\/Api/,'')
                    }
                }
            }
        }
    }

```

**Reply by bbhxwl** [reply]
> See [this video](https://www.youtube.com/watch?v=J4E5uYz5AY8) for thorough info on proxying an API :)

I understand, you are the author, my English is not good. Thank you.

**Reply by TheAlexLichter** [reply]
I'd suggest not to use vite's server proxy but nitro for that, either via routeRules or via custom endpoint.

**Comment by bbhxwl** [comment]
Nuxt is too powerful. I only use it as Vue3, but there are many routing conventions that reduce the use of imports
